numba/targets/iterators.py

```diff
@@ -61,6 +61,8 @@ def iternext_enumerate(context, builder, sig, args, result):
 
     with builder.if_then(is_valid):
         srcval = srcres.yielded_value()
+        if context.enable_nrt:
```

Comment:
Want to add a comment here for future visitors of this code path?

Reply:
done